cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/ArtifactInspector.java

```diff
@@ -188,6 +201,81 @@ public ArtifactClasses inspectArtifact(Id.Artifact artifactId, File artifactFile
   }
 
   /**
+   * we instantiate the plugin and invoke the plugin,
+   * we deserialize the request body into parameter type and pass it when invoking the plugin method and also
+   * optionally pass {@link EndpointPluginContext} to the method if its declared in the plugin method.
+   * @param pluginInstantiator to instantiate plugin instances.
+   * @param pluginClass call having the plugin method to invoke
+   * @param methodParameter parameter body
+   * @param endpointPluginContext endpoint plugin context that can optionally be passed to the method
+   * @throws IOException if there was an exception opening the jar file
+   * @throws InvalidArtifactException if the artifact is invalid
```

Comment:
this is not thrown, please remove. Also, methodName is not in the param list.  Please cleanup this javadoc